assets/src/dashboard/components/button/index.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types'; // import styled from 'styled-components';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { BUTTON_TYPES } from '../../constants';
+
+const StyledButton = styled.button`
+  align-items: center;
+  background-color: ${({ theme, type }) =>
+    type === BUTTON_TYPES.PRIMARY ? theme.colors.bluePrimary : 'transparent'};
+  color: ${({ theme }) => theme.colors.white};
+  cursor: pointer;
+  border: 1px solid transparent;
+  border-radius: ${({ theme, type }) =>
+    type === BUTTON_TYPES.PRIMARY ? theme.border.buttonRadius : 'none'};
+  display: flex;
+  height: 40px;
```

Comment:
No strong preference here but I find it more self-desciplined to have the height be determined by the content and make sure the content is indeed `40px` of height. The advantage is that you don't get clipped content and it's easier to spot styling errors when the entire button height changes.

Reply:
Yeah, I get that perspective. My thinking here is that the height can be overridden if it needs to be different through the parent component that's using the butotn. The figma docs all have this 40px and the button can expand for long text so i figure this is an ok default for now.